CHANGELOG.md

```diff
@@ -5485,6 +5485,7 @@
 [@wcmonty]: https://github.com/wcmonty
 [@nguyenquangminh0711]: https://github.com/nguyenquangminh0711
 [@chocolateboy]: https://github.com/chocolateboy
+[@RobinDaugherty]: https://github.com/RobinDaugherty
```

Comment:
I think that's not needed, as it will added automatically once we regen the changelog.